p2pd/main.go

```diff
@@ -71,6 +77,25 @@ func main() {
 		opts = append(opts, libp2p.NATPortMap())
 	}
 
+
+	if *relayEnabled {
+		var relayOpts []relay.RelayOpt
+		if *relayActive {
+			relayOpts = append(relayOpts, relay.OptActive)
+		}
+		if *relayHop {
+			relayOpts = append(relayOpts, relay.OptHop)
+		}
+		if *relayDiscovery {
+			relayOpts = append(relayOpts, relay.OptDiscovery)
+		}
+		opts = append(opts, libp2p.EnableRelay(relayOpts...))
+
+		if *autoRelay {
+			opts = append(opts, libp2p.EnableAutoRelay())
```

Comment:
that's not sufficient, you also need to pass the routing option and construct the DHT -- it's a more complex change.